src/discordant-stars/factions/zelian.js

```diff
@@ -143,21 +144,33 @@ const unitModifiers = [
   {
     // "space combat and abilities roll 1 additional die for each adjacent asteroid field",
     isCombat: true,
-    localeName: "unit.flagship.world-cracker",
-    localeDescription: "unit_modifier.desc.world-cracker",
+    localeName: "unit.flagship.world_cracker",
+    localeDescription: "unit_modifier.desc.world_cracker",
     owner: "self",
     priority: "adjust",
-    triggerUnitAbility: "unit.flagship.world-cracker",
+    triggerUnitAbility: "unit.flagship.world_cracker",
     filter: (auxData) => {
       return auxData.self.has("flagship");
     },
     applyAll: (unitAttrsSet, auxData) => {
       debugger;
-      const adjacentSystems = []; // get the adjacent systems somehow
-      const asteroidFields = adjacentSystems.filter(system => {
-        return system.anomalies.includes("asteroid field");
-      });
-      unitAttrsSet.get("flagship").raw.spaceCombat.dice += asteroidFields;
+      const adjacentTileNsids = world.TI4.Adjacency.getAdjacent(auxData.hex)
+        .map(hex => world.TI4.Hex.toPosition(hex))
+        .map(pos => world.TI4.getSystemTileObjectByPosition(pos))
+        .filter(tile => !!tile) // filter non-existing tiles (positions outside of the game setup)
+        .map(tile => tile.getTemplateMetadata());
```

Comment:
Better to stick to ObjectNamespace.getNsid(tile) just in case we ever change what's in the template metadata in the future.

Reply:
fixed in a new PR